### PR TITLE
oadp: turn off gcp for 4.12 master presubmits

### PR DESCRIPTION
**Pull request by kaovilai**
Signed-off-by: Tiger Kaovilai <tkaovila@redhat.com>

### CODE REVIEW

**Review by shubham-pampattiwar — APPROVED** [review]

File: ci-operator/jobs/openshift/oadp-operator/openshift-oadp-operator-master-presubmits.yaml [hunk 1]
```diff
@@ -1047,7 +1047,7 @@ presubmits:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )4.12-operator-e2e-azure,?($|\s.*)
   - agent: kubernetes
-    always_run: true
+    always_run: false
```
Comment (mateusoliveira43) [inline ci-operator/jobs/openshift/oadp-operator/openshift-oadp-operator-master-presubmits.yaml:1050]: I think we are missing `run_if_changed` in e2e aws, azure and unit test for 4.12 (for gcp not, as it should disabled for now)

Should we fix this in this PR?

Comment (kaovilai) [reply ci-operator/jobs/openshift/oadp-operator/openshift-oadp-operator-master-presubmits.yaml:1050]: Could do. sure

**Review by mateusoliveira43 — APPROVED** [review]

**Review by mateusoliveira43 — APPROVED** [review]

**Pull request merged**